src/components/PostPreview/PostPreview.style.ts

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
+
+export const PreviewWrapper = styled.div`
+  max-width: 600px;
+  width: 100%;
+  height: 150px;
+  margin: 0 auto;
+  padding: 15px 60px;
+  background-color: transparent;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-bottom: 0.5px solid ${({ theme }) => theme['greyLight']};
+`;
+
+export const PostHeaderWrapper = styled.div`
+  width: 100%;
+  height: 50px;
+  display: flex;
+  align-items: center;
+`;
+
+export const AvatarWrapper = styled.div`
+  width: 50px;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const PostInfoWrapper = styled.div`
+  width: calc(100% - 50px);
+  height: 50px;
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+`;
+
+export const UserNameWrapper = styled.div`
+  width: 100%;
+  height: 50%;
+  font-weight: bold;
+  font-size: 16px;
+  display: flex;
+  align-items: center;
+`;
+
+export const PostDetailInfoWrapper = styled.div`
+  width: 100%;
+  height: 50%;
+  color: ${({ theme }) => theme['greyLight']};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.color.greyLight;
```
color프로바이더를 분리했기 때문에 theme.color.greyLight형식으로 사용해야 합니다!
빨리 공통 스타일 브랜치가 머지 되야 할텐데ㅠ
머지 되면 pull받고 수정해주세요!

Reply:
다른 컴포넌트들도 수정해야겠군요 ㅠㅠ

Reply:
제가 기존에 사용하던 컴포넌트는 다 수정해놨습니다!